features/data_converter/json_protobuf/feature.go

```diff
@@ -0,0 +1,64 @@
+package json_protobuf
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/gogo/protobuf/jsonpb"
+	pb "go.temporal.io/features/features/data_converter"
+	"go.temporal.io/features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/converter"
+	"go.temporal.io/sdk/workflow"
+)
+
+var EXPECTED_RESULT = []byte{0xde, 0xad, 0xbe, 0xef}
+
+var Feature = harness.Feature{
+	Workflows:   Workflow,
+	CheckResult: CheckResult,
+	ClientOptions: client.Options{
+		DataConverter: converter.NewCompositeDataConverter(
```

Comment:
You don't need a custom data converter, this feature is on by default in our SDKs

Reply:
I added a comment to clarify that ProtoJSONPayload has priority over ProtoPayload in the default converter, and deleted it.